src/sidebar/components/Annotation/AnnotationBody.js

```diff
@@ -48,8 +52,25 @@ function AnnotationBody({ annotation, settings }) {
 
   const textStyle = applyTheme(['annotationFontFamily'], settings);
 
+  // Determine whether tags in the list of tags should be linked to tag
+  // search results
+  const shouldLinkTags = useMemo(
+    // Show a link if the authority of the user is not 3rd party
```

Comment:
I think these two comments could be omitted. The code seems clear enough to me on its own.